web_src/css/base.css

```
@@ -2796,21 +2766,16 @@ table th[data-sortt-desc] .svg {
   height: 15px;
 }
 
-@media (max-width: 767px) {
-  .ui.stackable.menu:not(.no-vertical-tabs) {
-    overflow-y: hidden;
-    overflow-x: auto;
-    flex-direction: row;
-    flex-wrap: nowrap !important;
-  }
-  .ui.stackable.menu:not(.no-vertical-tabs) .item {
-    width: initial !important;
-  }
-  .ui.stackable.menu:not(.no-vertical-tabs) > .dropdown.item {
-    position: initial;
-  }
-  .ui.stackable.menu:not(.no-vertical-tabs) .menu {
-    flex-direction: row;
+.g-menu-stackable-scrollable {
+  overflow-y: hidden;
+  overflow-x: auto;
+}
+
+@media only screen and (max-width: 767.98px) {
```

```suggestion
/* breakpoint matches Fomantic */
@media (max-width: 767.98px) {
```

In our code, we don't use the `screen` restriction anywhere else besides here, so let's remove it for consistency.

---

But what does `breakpoint matches Fomantic ` mean?

---

```suggestion
@media (max-width: 767.98px) {
```

---

All our code has `@media (max-width: 767px)`, all fomantic code has `@media only screen and (max-width: 767.98px)`, that is the difference I meant. The difference does not mean much but there is a tiny range where ours and fomantic's breakpoint do not match.

---

It does look fomantic hardcodes this and has not made it configurable:

https://github.com/fomantic/Fomantic-UI/search?p=2&q=767.98px